lib/queue.rb

```diff
@@ -1,31 +1,32 @@
+require './linked_list'
+
 class Queue
```

Comment:
This works, but it's not implemented using a circular buffer, so it's not what was asked for.